doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -18,7 +18,7 @@
 
 ### Workflow
 
- * Example [#1](https://github.com/larry-robotics/iceoryx2/issues/1)
+ * add `cargo audit` for security vulnerability checking in dependencies [#49](https://github.com/eclipse-iceoryx/iceoryx2/issues/49)
```

Comment:
```suggestion
 * add `cargo audit` for security vulnerability checking in dependencies [#48](https://github.com/eclipse-iceoryx/iceoryx2/issues/48)
```

Please always link to the actual issue instead of the PR :)

Reply:
@hydroid7 whoopsie my mistake I mixed up the PR number and the issue number